mono/metadata/Makefile.am

```diff
@@ -162,6 +162,8 @@ common_sources = \
 	decimal-ms.c		\
 	decimal-ms.h		\
 	domain-internals.h	\
+	ee.c			\
```

Comment:
Why are these files in metadata/ ?

Reply:
it follows the same idea as `metadata/profiler.c`

Reply:
It should be in mini/ imho, if we really need this, since its not used by metadata/.